### DISCUSSION TITLE
how to list only users that belong to the tenant

### DISCUSSION THREAD

**Discussion by joaopola** [filamentphp/filament#13384] [Help]
### Package

Table builder

### Package Version

v3.2

### How can we help you?

I am not using any rules and permissions packages, but I would like to list only users with the tenant_id of the logged in tenant.

**Comment by micraux** [comment]
Hello,

You can [customize the query](https://filamentphp.com/docs/3.x/panels/resources/getting-started#customizing-the-resource-eloquent-query), something like:

```php
use Filament\Facades\Filament;

-----

public static function getEloquentQuery(): Builder
{
    return parent::getEloquentQuery()->where('tenant_id', Filament::getTenant()->tenant_id);
}
```

Does it help you?
